docs/archiver/archiver-guide-mainnet-sync.md

```diff
@@ -0,0 +1,205 @@
+# Running a QuarkChain L2 Node to Derive Blocks from EthStorage Mainnet Archive Service
+
+## Table of Contents
+1. **[Introduction](#introduction)**
+
+2. **[Running L2 Node](#running-l2-node)**
+   - 2.1 [Initializing op-geth](#preparing-op-geth)
+   - 2.2 [Starting op-geth](#starting-op-geth)
+   - 2.3 [Initializing op-node](#preparing-op-node)
+   - 2.4 [Starting op-node](#starting-op-node)
+
+3. **[Verifying the Derivation Process](#verifying-the-derivation-process)**
+   - 3.1 [Observing the Logs](#observing-the-logs)
+     - 3.1.1 [op-geth logs](#op-geth-logs)
+     - 3.1.2 [op-node logs](#op-node-logs)
+     - 3.1.3 [es-node logs](#es-node-logs)
+   - 3.2 [Comparing Blocks](#comparing-blocks)
+
+4. **[Conclusion](#conclusion)**
+
+
+## Introduction
+
+This guide walks you through spinning up a QuarkChain L2 node on mainnet using the EthStorage archive service.
+
+QuarkChain L2, built on OP Stack, publishes batches as blobs to its [BatchInbox contract](https://etherscan.io/address/0xf62e8574B92dc8764c5Ad957b5B0311595f5A3f9) on Ethereum mainnet. EthStorage stores those blobs as a long-term data-availability layer.
+
+The objective is to verify that the L2 node can derive blocks correctly from blobs fetched via EthStorage.
```

Comment:
Maybe we should be more clear whether the purpose here is:
1. walks you through spinning up a QuarkChain L2 node on mainnet using the EthStorage archive service; or
2. The objective is to verify that the L2 node can derive blocks correctly from blobs fetched via EthStorage.

as they sound like two different purpose to me.

(I guess 2 is the real purpose?)